dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.ts

```diff
@@ -17,13 +18,16 @@ export class PlatformStatusWidgetComponent extends Widget {
     super(wS);
   }
 
+  public initWidget(widgetConfig: WidgetConfig) {
```

Comment:
Do you have to override this method? As you are just calling super, there should be no need to override it.